File: multiples3or5/multiples3or5.go
package main

import (
	"fmt"
)

func multiplesOf3or5(endpoint int) int {
	sum := 0
	for i := 3; i < endpoint; i++ {
		if i%3 == 0 {
			sum += i
		} else if i%5 == 0 && i >= 5 {
			sum += i
		}
	}
	return sum
}

func main() {
	num := 1000
	fmt.Printf("the sum of all multiples of 3 and 5 between zero and %d is %d", num, multiplesOf3or5(num))
}

File: multiples3or5/multiples3or5.py
def multiples3or5(num):
    sum = 0
    for i in range(3, num):
        if i % 3 == 0:
            sum += i
        elif (i % 5 == 0) and i >= 5:
            sum += i
    return sum

num = 1000
print(f'Sum of all multiples of 3 and 5 from 0 to {num} is {multiples3or5(num)}')